avocado/core/runners/avocado_instrumented.py

```diff
@@ -41,8 +42,15 @@ def _create_params(runnable):
             paths = runnable.variant['paths']
             return tree_nodes, paths
 
+    def _is_running_with_coverage(self):
+        # Coverage.py sets an environment variable, COVERAGE_RUN to indicate
+        # that your code is running under coverage measurement.
+        if os.environ.get('COVERAGE_RUN', False):
+            return True
+        return False
+
     @staticmethod
-    def _run_avocado(runnable, queue):
+    def _run_avocado(runnable, queue, run_coverage=False):
```

Comment:
Given the simplicity of the check, I don't think we need a new parameter here, instead, just perform the check where needed (line 82).

Reply:
@clebergn, line 82 its is running inside the subprocess. Yes, currently we export the env variables to our subprocess, but I decided left this decision outside to give us flexibility for future changes, in that way we don't need to preserve this variable if we change the way to spawn the subprocess. Maybe too much paranoia on my side.

Reply:
> @clebergnu, line 82 its is running inside the subprocess. Yes, currently we export the env variables to our subprocess, but I decided left this decision outside to give us flexibility for future changes, in that way we don't need to preserve this variable if we change the way to spawn the subprocess. Maybe too much paranoia on my side.

Yeah... I don't think adding complexity to the current code is a good enough justification for something that may not be needed in the future.

Reply:
> Yeah... I don't think adding complexity to the current code is a good enough justification for something that may not be needed in the future.

Same thought.